src/seedu/addressbook/data/person/Name.java

```diff
@@ -39,6 +41,10 @@ public static boolean isValidName(String test) {
     /**
      * Retrieves a listing of every word in the name, in order.
      */
+    public List<String> getFindWordsInName() {
+        return Arrays.asList(lowerCaseName.split("\\s+"));
+    }
+
     public List<String> getWordsInName() {
```

Comment:
Deprecated method
![image](https://user-images.githubusercontent.com/23214709/36268597-dbcc2552-12b1-11e8-8be0-1e4b476d61c4.png)
**getWordsInName** method was only called by **FindCommand** class but now **getFindWordsInName** is called instead of **getWordsInName**.